addons/stock/wizard/product_replenish.py

```diff
@@ -38,10 +38,12 @@ def default_get(self, fields):
                 product_tmpl_id = product_id.product_tmpl_id
                 res['product_tmpl_id'] = product_id.product_tmpl_id.id
                 res['product_id'] = product_id.id
+                res['quantity'] = abs(product_id.virtual_available)
```

Comment:
why abs ?

Reply:
Hi, 
we need to display product quantity in positive value,  there might be some case when forecast quantity in negative number that is why I took abs.